com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
>Is it just the view cannot be located, or that no such view currently exists for a job/instance

From the client side, I'm not sure if there's a way to tell the difference.

If no such view exists for an instance, then we won't be able to locate it <==> If we can't locate it, then no such view exists.